.github/workflows/build.yml

```diff
@@ -1,43 +1,52 @@
-name: build
+name: "build"
 
 on:
   push:
-    branches: [ develop,master ]
+    branches: [ develop ]
```

Comment:
when release a version, it need push all the commit from develop branch to master branch.

Reply:
the `test.yml` will replace the `build.yml` to execute the tests.

Reply:
There will be more tests for the `master` branch in the future, such as `test-redis.yml`、`test-mysql-driver.yml`、...